index.html

```diff
@@ -0,0 +1,262 @@
+                            <p id="pcareer">Back End Dev</p>
+                        </section>
+                        <img id="counter">
+                        <section id="year_label">
+                            <p id="pcareer">2015</p>
+                        </section>
+                    </section>
+                </section>
+                <p id="sub_text">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <section id="tags">
+                    <section id="tag1">
+                        <p id="thtml">html</p>
+                    </section>
+                    <section id="tag2">
+                        <p id="tcss">css</p>
+                    </section>
+                    <section id="tag3">
+                        <p id="tjavascript">javascript</p>
+                    </section>
+                </section>
+                <div id="action">
+                    <button id="normal_button">
+                        <p id="see_project">See Project</p>
+                    </button>
+                </div>
+            </section>
+        </section>
+        <section id="about_myself">
+            <h1>About Myself</h1>
+            <p>Hello I'm a software developer! I can help you build a product, feature or website.
+                Look through some of my work and exoerience!
+                If you like what you see and have a project you need coded, don't hesitate to contact me.
+            </p>
+            <a href="#">LET’S CONNECT</a>
+            <ul class="social-links">
+                <li>
+                    <a href="#"><img src="images/twitter.svg" alt="Twitter"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/linkedinicon.svg" alt="LinkedIn"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/Vector (1).svg" alt="Angel List"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/github.svg" alt="Github"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/Vector.svg" alt="Meduim"></a>
+                </li>
+            </ul>
+
+        </section>
+        <button id="normal_button1">
+            <p id="resume">Get my resume</p>
+        </button>
+        <ul id="skill_list">
+            <li id="languages">
+
+                <section id="langbox">
+                    <p id="lang_text">Languages</p>
+                    <img id="down_arrow" src="images/downarrow.svg">
+                </section>
+                <ul>
+                    <li><a id="langicon" href="#"><img src="images/javascript.svg" alt="javascript"></a></li>
+                    <li><a id="langicon" href="#"><img src="images/html.svg" alt="html"></a></li>
+                    <li><a id="langicon" href="#"><img src="images/css.svg" alt="css"></a></li>
```

Comment:
[ ] Great at implementing the about-myself section design, but some issues to figure out in your design. As much as possible, stick to the Figma template you choose. Make sure of the alignment of the section, like the CSS property margin and padding as the designed template.

| Figma template design  | Your design|
| ------------- | ------------- |
|![Screenshot from 2022-10-29 16-42-58](https://user-images.githubusercontent.com/65129228/198836502-1a3f025c-3443-4b16-9627-8a0d7ce78949.png)| ![Screenshot from 2022-10-29 16-47-10](https://user-images.githubusercontent.com/65129228/198836526-a194957a-77ef-4c67-b76c-5d47678e4732.png)|